HousingSearchListener/V1/Domain/Tenure/TempAccommodationInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace HousingSearchListener.V1.Domain.Tenure
+{
+    public class TempAccommodationInfo
+    {
+        public string BookingStatus { get; set; }
```

Comment:
This model being created here seems to already exist within the shared package:

https://github.com/LBHackney-IT/housing-search-shared/pull/73/files#diff-a9de02cd0a25aa301bc60b68f2499d9756a1ad36d6744531ba4fe6076cd45133R5-R217

Looking at the file bellow (`HousingSearchListener/V1/Domain/Tenure/TenureInformation.cs`) - it seems like its utilising this model that's being created here instead of one from shared package.

Perhaps, we could point to the one from shared packaged instead?

Reply:
@Duslerke that's a good point. There seems to be a lot of duplication of models here that are alreade available in shared. In the interest of time for this ticket I'll merge this as is, so it follows the existing pattern, but we can look at tidying things up in the next ticket where we handle the assigned officer details in this same model. Need to take a closer look at  why these models are in two different places.